### PR TITLE
Update doc/user-guides/secure-protect-connect.md

### PR DESCRIPTION
**Pull request by maleck13**
Co-authored-by: Jason Madigan <4467+jasonmadigan@users.noreply.github.com>

number and fix link

remove uneeded resolve

remove unneeded var

### CODE REVIEW

**Review by jasonmadigan — COMMENTED** [review]
Looks to have addressed all of @guicassolato's feedback

**Review by maleck13 — COMMENTED** [review]
looks good, but I can't approve it

**Review by guicassolato — APPROVED** [review]

File: doc/user-guides/secure-protect-connect.md [hunk 1]
```diff
@@ -1,35 +1,39 @@
-# Kuadrant Quick Start
+# Secure, Protect and Connect services with Kuadrant
 
 ## Pre-requisites
 
-- Completed the [single cluster quick start](https://docs.kuadrant.io/getting-started-single/)
+- Completed the [Single-cluster Quick Start](https://docs.kuadrant.io/getting-started-single-cluster/)
```
Comment (eguzki) [inline doc/user-guides/secure-protect-connect.md:5]: This link returns `404`

Comment (eguzki) [reply doc/user-guides/secure-protect-connect.md:5]: Ok, I guess this https://github.com/Kuadrant/docs.kuadrant.io/pull/44 needs to be merged

**Pull request merged**